apps/swirl-docs/src/components/ComponentPreview/CodePreview.tsx

```diff
@@ -69,7 +69,7 @@ export const CodePreview: FunctionComponent<CodePreviewProps> = ({
             </button>
           </CopyToClipboard>
         </div>
-        <pre className="cursor-text pt-space-8 pr-space-24 pb-16 pl-space-24">
+        <pre className="cursor-text pt-space-8 pr-space-24 pb-16 pl-space-24 whitespace-pre-wrap">
```

Comment:
**suggestion:** I think it's more common to not wrap code examples and instead give the container a horizontal scrollbar.